Wanderer/www/modules/dc-list-reputations/page.html

```diff
@@ -0,0 +1,14 @@
+﻿<link rel="stylesheet" type="text/css" href="modules/dc-list-reputations/stylesheet.css">
+<div id="whole_thing">
+    <div class="reputation" data-ng-repeat="reputation in module.reputations track by $index">
+        <input type="text" data-ng-model="module.reputations[$index]"
+                           data-ng-model-options="{updateOn: 'blur default',
+                                                   debounce: {
+                                                      'default': 500,
+                                                      'blur': 0  
+                                                       }
+                                                   }"></input>
```

Comment:
Does this fix [the lag issue](https://github.com/Colinwielga/Wanderer-Cordova/issues/8)?